bin/webdrivers/chrome/README.md

```diff
@@ -4,39 +4,81 @@ Chrome webdriver for Selenium Server Standalone, the browser automation framewor
 
 ## Version
 Release numbers are synchronised with the Selenium versions.
-Version of this release is *v2.23.0*
+Version of this release is *v2.53.1.1*
 
 ## Installation
 
-#### Inside your robofile
+### Inside your robofile (RoboFile.php in the root directory)
 
-Add the following line of code after the jar file
+Add the following line of code after the call up of the jar file:
 
-For Windows:
+####For Windows:
 ```
 -Dwebdriver.chrome.driver=.\\vendor\\joomla-projects\\selenium-server-standalone\\bin\\webdrivers\\chrome\\chromedriver.exe
 ```
 
-For Mac:
+####For Mac:
 ```
--Dwebdriver.chrome.driver=.\\vendor\\joomla-projects\\selenium-server-standalone\\bin\\webdrivers\\chrome\\chromedriver_mac
+-Dwebdriver.chrome.driver=./vendor/joomla-projects/selenium-server-standalone/bin/webdrivers/chrome/chromedriver_mac
 ```
 
-For Linux:
+####For Linux:
 ```
 32bit
--Dwebdriver.chrome.driver=.\\vendor\\joomla-projects\\selenium-server-standalone\\bin\\webdrivers\\chrome\\chromedriver_linux_32
+-Dwebdriver.chrome.driver=./vendor/joomla-projects/selenium-server-standalone/bin/webdrivers/chrome/chromedriver_linux_32
 ```
 ```
 64bit
--Dwebdriver.chrome.driver=.\\vendor\\joomla-projects\\selenium-server-standalone\\bin\\webdrivers\\chrome\\chromedriver_linux_64
+-Dwebdriver.chrome.driver=./vendor/joomla-projects/selenium-server-standalone/bin/webdrivers/chrome/chromedriver_linux_64
 ```
 
-Change the browser setting on /tests/acceptance.suite.yml to:
+If use Linux on a 64bit machine your robofile.php should look like this:
+```
+  ...
+	/**
+	 * Runs Selenium Standalone Server.
+	 *
+	 * @since   __DEPLOY_VERSION__
+	 *
+	 * @return  void
+	 */
+	public function runSelenium()
+	{
+		if (strtoupper(substr(PHP_OS, 0, 3)) !== 'WIN')
+		{
+			$this->_exec($this->testsPath . "vendor/bin/selenium-server-standalone -Dwebdriver.chrome.driver=./vendor/joomla-projects/selenium-server-standalone/bin/webdrivers/chrome/chromedriver_linux_64 >> selenium.log 2>&1 &");
+		}
+		else
+		{
+			$this->_exec("START java.exe -jar .\\tests\\codeception\\vendor\\joomla-projects\\selenium-server-standalone\\bin\\selenium-server-standalone.jar");
```

Comment:
@astridx Can you adjust the windows path too, it should be the following:

-Dwebdriver.chrome.driver=.\\vendor\\joomla-projects\\selenium-server-standalone\\bin\\webdrivers\\chrome\\chromedriver.exe